content/guides/spring/spring-route-authorization-testing.md

```diff
@@ -0,0 +1,550 @@
+---
+draft: false
+date: 2021-05-05T10:40:17-06:00
+tags: ["spring","testing", "spring security"]
+title: "Spring Route Authorization Testing"
+description: >
+    Dynamic testing of your Spring Security route authorizations
+date: "2021-05-12"
+topics:
+- Spring
+- Testing
+tags:
+- Spring
+- Spring Security
+- Spring Testing
+# Author(s)
+team: 
+- Toby Rumans
+---
+
+Spring Security enables us to lock down routes in applications to particular roles. 
+
+Testing route authorizations are often manual processes, that are prone to human error and often have complicated setup. 
+
+The following are dynamic tests that ensure your route authorization configuration is up-to-date, and has 100 percent coverage.
+
+**Code examples** 
+
+<https://github.com/tobocop/spring-route-authorization-testing>
```

Comment:
Is the code that's presented inline in the article enough to demonstrate this pattern? If so, it might be simplest to avoid linking to your Github until we have a better location for external code samples.

Reply:
It's technically enough though the code presented doesn't have any implementation code to build off of. Maybe that should be added though as part of the guide. Mostly what I wanted from the repo is so people could look at dependencies and the gradle file and all that stuff even if it's simple.

Reply:
Talked with Zac over slack. Leaving this in for now until a better alternative is provided.